src/Styleguide/Components/ArtistInsight.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box, Flex, Link, Sans } from "@artsy/palette"
+import { BlueChip } from "Assets/Icons/BlueChip"
+import React from "react"
+
+export interface ArtistInsightProps {
+  type: string
+  label: string
+  value?: string
+  entities?: ReadonlyArray<string>
+}
+
+export class ArtistInsight extends React.Component<ArtistInsightProps> {
+  state = {
+    expanded: false,
+  }
+
+  renderEntities() {
+    const { entities } = this.props
+
+    if (entities.length > 1) {
+      if (this.state.expanded) {
+        return (
+          <Sans size="2" verticalAlign="top" color="black60">
+            {entities.join(", ")}.{" "}
+            <Link onClick={() => this.setState({ expanded: false })}>
+              Show less
+            </Link>
+          </Sans>
+        )
+      } else {
+        return (
+          <Sans size="2" verticalAlign="top" color="black60">
+            {entities[0]}, and{" "}
+            <Link onClick={() => this.setState({ expanded: true })}>
+              {entities.length - 1} more
+            </Link>
+          </Sans>
+        )
+      }
+    } else {
+      return (
+        <Sans size="2" verticalAlign="top" color="black60">
+          {entities[0]}
+        </Sans>
+      )
+    }
+  }
+
+  render() {
+    const { label, value, entities } = this.props
+
+    return (
+      <Flex mt={1} alignItems="top" width={260}>
+        <Flex mr={1} flexGrow={0} alignItems="top">
+          <BlueChip />
```

Comment:
cool, this can be passed in by the client or inferred by the 'type' of insight or whatever